src/components/course/course-header/CourseImportantDates.jsx

```diff
@@ -88,11 +89,12 @@ const CourseImportantDates = () => {
   // Match soonest expiring assignment to the corresponding course start date from course metadata
   let soonestExpiringAllocatedAssignmentCourseStartDate = null;
   if (soonestExpiringAssignment) {
-    soonestExpiringAllocatedAssignmentCourseStartDate = courseMetadata.availableCourseRuns.find(
+    const soonestExpiringAllocatedAssignment = courseMetadata.availableCourseRuns?.find(
```

Comment:
[nit/curious]: when is `availableCourseRuns` not set where the `?.` is needed when it wasn't needed before?